app/main.py

```diff
@@ -30,32 +40,42 @@
     allow_headers=["*"],
 )
 
-
-# Dependency
-def get_db():
-    db = SessionLocal()
-    try:
-        yield db
-    finally:
-        db.close()
+@app.post("/token", response_model=schemas.Token)
+async def login_for_access_token(form_data: OAuth2PasswordRequestForm = Depends(), db: Session = Depends(get_db)) -> dict:
+    user = authenticate_user(db, form_data.username, form_data.password)
+    if not user:
+        raise HTTPException(
+            status_code=status.HTTP_401_UNAUTHORIZED,
+            detail="Incorrect username or password",
+            headers={"WWW-Authenticate": "Bearer"},
+        )
+    
+    if settings.JWT_EXPIRE_MINUTES is None:
```

Comment:
no if de cima vc fez correto, mas nesse está o is None, é  melhor usar o not,
e é bem importante manter um padrão no código!